scapy/layers/bluetooth.py

```diff
@@ -1268,6 +1279,8 @@ class HCI_LE_Meta_Long_Term_Key_Request(Packet):
                    XLEShortField("ediv", 0), ]
 
 
+conf.l2types.register(DLT_BLUETOOTH_LINUX_MONITOR, BT_Mon_Pcap_Hdr)
```

Comment:
Can you move that next to the other `conf.l2types`?

Reply:
Grouped! Permission request to move all `conf.l2types` to the bottom of `bind_layers` definitions. Is that desirable to keep a better overall ordering? Thanks

Reply:
Propably not necessary